docs/index.rst

```diff
@@ -117,15 +117,22 @@ Nuts and bolts
    logging
    warnings
    utils/index
+
+{% if is_development %}
```

Comment:
I wonder if it makes sense to just move these two items to the developer documentation section?